edb/lang/edgeql/parser/grammar/lexer.py

```diff
@@ -174,6 +174,14 @@ class EdgeQLLexer(lexer.Lexer):
                 (?P=Q)
              '''),
 
+        Rule(token='BADSCONST',
```

Comment:
Please add a comment explaining that this will match any string-like thing that wasn't matched by string regexps above